src/aapns/models.py

```diff
@@ -37,11 +37,11 @@ def maybe_localized(
 @attr.s
 class Alert:
     body: Union[str, Localized] = attr.ib(
-        validator=instance_of_validator(str, Localized)
+        validator=instance_of_validator((str, Localized))
     )
     title: Optional[Union[str, Localized]] = attr.ib(
         default=None,
-        validator=attr.validators.optional(instance_of_validator(str, Localized)),
+        validator=attr.validators.optional(instance_of_validator((str, Localized))),
```

Comment:
Would something like `lambda _,_,value: value is None or isinstance(value, (str, Localized))` be a bit easier to read?
(possible with `or raise_me_a_typeerror()` added to comply with the validator API).

Reply:
We can argue about readability but the problem is it would break mypy :(